authority/authorize_test.go

```diff
@@ -296,9 +330,10 @@ func TestAuthority_authorizeRevoke(t *testing.T) {
 			raw, err := jwt.Signed(sig).Claims(cl).CompactSerialize()
 			assert.FatalError(t, err)
 			return &authorizeTest{
-				auth: a,
-				opts: &RevokeOptions{OTT: raw},
-				err:  errors.New("authorizeRevoke: token subject cannot be empty"),
+				auth:  a,
+				token: raw,
+				err:   errors.New("authority.authorizeRevoke: jwk.AuthorizeRevoke: jwk.AuthorizeToken; jwk token subject cannot be empty"),
```

Comment:
I agree this would be nice. Mariano commented somewhere above that while these changes are an improvement to the way we handle errors, there is still a lot more we can do - both in handling and in testing the errors.